src/main/java/frc/robot/commands/ShootOneBallCommand.java

```diff
@@ -27,7 +28,7 @@ public void initialize() {
 
   @Override
   public void execute() {
-    if (m_shooterSubsystem.getShooterSpeed() >= 0.96) {
+    if (m_shooterSubsystem.getFlywheelRPM() >= 0.96) {
```

Comment:
more of a content thing but 0.96 is not in RPM and should definitely be changed to 5000 or something. also, i'd say that whatever number we choose should go in constants as our desired shooter RPM.

Reply:
Yeah, I think the number 0.96 is wrong.  But also, what should that Constant be called?

Reply:
I'm calling it SHOOTER_READY_RPM or FLYWHEEL_READY_RPM